Sources/App/configure.swift

```diff
@@ -7,13 +7,16 @@ public func configure(_ app: Application) throws {
     // uncomment to serve files from /Public folder
     // app.middleware.use(FileMiddleware(publicDirectory: app.directory.publicDirectory)){{#fluent}}
 
+    {{#fluent.db.is_postgres}}
```

Comment:
Sorry, looks like this extra line snuck into one of my suggestion comments, once it's gone this is ready for merge.
```suggestion
```